Sources/Client/Model/Channel/Channel.swift

```diff
@@ -438,6 +438,15 @@ public struct MessageResponse: Decodable {
     public let message: Message
     /// A reaction.
     public let reaction: Reaction?
+    /// Owner channel of this message. Only available when a message is queried by its `id`.
+    @NestedKey
+    public var channel: Channel?
```

Comment:
This is a get-only property, but I can make it `let` because of the property wrapper. So no worries here

Reply:
Maybe you can add `internal(set)`

Reply:
There's no need, it acts like a computed property so trying to set results in a compiler error